docs/docs/guide/03-blog/02-connect-blockchain.md

```diff
@@ -93,15 +93,18 @@ func main() {
 	// account `alice` was initialized during `ignite chain serve`
 	accountName := "alice"
 
-	// get account from the keyring by account name and return a bech32 address
-	address, err := cosmos.Address(accountName)
+	// get account from the keyring by account name
+	account, err := cosmos.Account(accountName)
 	if err != nil {
 		log.Fatal(err)
 	}
+	
+	// get the bech32 version of the account with proper prefix
+	address := cosmos.Address("cosmos")
```

Comment:
Hi @dmikey! I am curious, why do you want to make the "cosmos" prefix explicit?

The [cosmos.Address()](https://github.com/ignite/cli/blob/develop/ignite/pkg/cosmosclient/cosmosclient.go#L203) call won't work with the "cosmos" argument because it is not a valid account name.

Reply:
@jeronimoalbi apologies, corrected, I left that line there as I will feeling my way through getting this to work with my altered chain. This should be `account.Address('prefix')` where `account` was retrieved by name from the `cosmosclient` above. Thanks for pointing this out, corrected!

Reply:
Hey @dmikey! I believe you could accomplish what you want using something like:

```go
// Prefix to use for account addresses.
//
// The prefix is assigned in the `app/app.go` file
// using the `AccountAddressPrefix` constant or using
// the `--address-prefix` flag when scaffolding a new
// chain with Ignite CLI.
addressPrefix := "blog"

// Create a Cosmos client instance
cosmos, err := cosmosclient.New(
	context.Background(), 
	cosmosclient.WithAddressPrefix(addressPrefix),
)
if err != nil {
	log.Fatal(err)
}

// Account `alice` was initialized during `ignite chain serve`
accountName := "alice"

// Get account from the keyring
account, err := cosmos.Account(accountName)
if err != nil {
	log.Fatal(err)
}

// Define a message to create a post
msg := &types.MsgCreatePost{
	Creator: account.Address(addressPrefix),
	Title:   "Hello!",
	Body:    "This is the first post",
}

// ...
```

Does it work for you?

Reply:
Absolutely! I think that's a good compromise.

Reply:
There is another way that uses the cosmos SDK config but it should not be "sealed" for the solution to work:

```go
import (
	// ...

	sdk "github.com/cosmos/cosmos-sdk/types"
)

// ...

// Prefix for account addresses
addressPrefix := "blog"
pubKeyPrefix := addressPrefix + "pub"

// Set the global account address prefix.
// Note: Don't seal the config by calling `cfg.Seal()`
cfg := sdk.GetConfig()
cfg.SetBech32PrefixForAccount(addressPrefix, pubKeyPrefix)

// Create a Cosmos client instance
cosmos, err := cosmosclient.New(
	context.Background(), 
	cosmosclient.WithAddressPrefix(addressPrefix),
)
if err != nil {
	log.Fatal(err)
}

// Account `alice` was initialized during `ignite chain serve`
accountName := "alice"

// Get the account address
address, err := cosmos.Address(accountName)
if err != nil {
	log.Fatal(err)
}

// Define a message to create a post
msg := &types.MsgCreatePost{
	Creator: address.String(), // <- address with "blog" prefix
	Title:   "Hello!",
	Body:    "This is the first post",
}

// ...
```